ramp-database/ramp_database/cli.py

```diff
@@ -189,6 +191,80 @@ def add_submission(config, event, team, submission, path):
                                          path)
 
 
+@main.command()
+@click.option("--config", default='config.yml', show_default=True,
+              help='Configuration file YAML format containing the database '
+              'information')
+@click.option("--config-event", required=True,
+              help="""Path to configuration file YAML format
+              containing the database information, eg config.yml""")
+@click.option('--from-disk', default=False,
+              help="""If True the event and all the related submissions
+                      will be removed from the database and from the disk;
+                      if set to False they will only be removed from
+                      the database""")
+@click.option('--force', default=False,
+              help="""If True the event will be removed from the disk
+                      (if it exists) even if it is not found
+                      in the database.
+                      This option must be used with --from-disk
+                      option to work""")
+def delete_event(config, config_event, from_disk, force):
+    """Delete event."""
+    config = read_config(config)
+
+    # check if the event_config dir exists
+    if not os.path.isfile(config_event):
+        click.secho("No such file: {}".format(config_event),
+                    err=True, fg='red')
+    else:
+        # read the event_name
+        config_event_params = read_config(config_event)
+        event_name = config_event_params['ramp']['event_name']
+        event_dir = os.path.dirname(config_event)
+
+        # remove event from the database
+        with session_scope(config['sqlalchemy']) as session:
+            # is event in the database?
+            db_event = event_module.get_event(session, event_name)
+            if db_event:
+                event_module.delete_event(session, event_name)
+
+                if from_disk:
+                    # remove submissions dir from disk if it exists
+                    path_to_submissions = db_event.path_ramp_submissions
+                    if os.path.exists(path_to_submissions):
+                        shutil.rmtree(path_to_submissions)
+                        click.echo("removed directory " +
+                                   "{}".format(path_to_submissions))
+                    else:
+                        click.secho("No such submissions directory: " +
+                                    "{}".format(path_to_submissions),
+                                    fg='red', err=True)
+            else:
```

Comment:
we need before this else the
```python
elif from_disk and force:
    # forcing to remove from the disk.
```
In this case we don't get any message that the event is not in the database.